### PR TITLE
DR-36: scripts to create the postgres database

### PR DESCRIPTION
**Pull request by ahaessly**
scripts and markdown to install and create db locally

### CODE REVIEW

File: DATABASE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+### Install
+
+Install [postgres server version 9.6](https://github.com/PostgresApp/PostgresApp/releases/download/v2.2/Postgres-2.2-9.5-9.6-10-11.dmg)
+
+This version defaults to starting postgres server version 11. To change to version 9.6:
```
Comment (ddietterich) [inline DATABASE.md:5]: Why version 9.6?

File: db/create-jade-db [hunk 2]
```diff
@@ -0,0 +1,3 @@
+CREATE USER jade WITH PASSWORD 'jadepasswd';
```
Comment (ddietterich) [inline db/create-jade-db]: I suggest avoiding use of the project name in the code. In my experience, project names often need to be changed. For example, another open source project or commercial product named Jade that we collide with.

My preference would be to name the user "data-repo-manager" and the database "data-repo-metadata" or similar.

File: DATABASE.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+2. Stop the server
+2. Click the icon on the lower left to show the servers
+3. Select Postgres11 and then -
+4. Select + and choose version 9.6
+5. Start the server
+
+
+### Create the jadedb and user
+
+`pslq -f db/create-jade-db`
```
Comment (ddietterich) [inline DATABASE.md]: Did you mean `psql`?

**Review by ddietterich — CHANGES_REQUESTED** [review]

File: DATABASE.md (hunk 1 above)
Comment (ahaessly) [reply DATABASE.md:5]: that is the latest supported by google could sql

File: DATABASE.md (hunk 3 above)
Comment (ahaessly) [reply DATABASE.md]: of course! :)

**Review by thathert — APPROVED** [review]
I've de-jade-ified the names and ran this locally -- works for me. @ddietterich can you re-review?

**Review by ddietterich — APPROVED** [review]

**Pull request merged**